specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -133,9 +122,3 @@ Consider at least the below topics but you do not need to cover those that are n
 <!--
 Optionally, describe what are the implementation stories (eventually we'd create github issues out of them).
 -->
-
-## Alternatives
-<!--
-Optionally, describe what alternatives has been considered.
-Keep it short - if needed link to more detailed research document.
--->
```

Comment:
nit: here is a good place to elaborate on code quality and security scanners, and their adoption.

Reply:
Alternative security scanners? 

I would think that is too low level. 
I would think alternatives here would be auto generated GUI from swagger docs etc... 

e.g alternatives to having a frontend GUI as opposed to alternatives for low level implementation details.

Reply:
So, what you say is that the Frontend alternatives are Control Service API and VDK CLI (API client), is that so?